### DISCUSSION TITLE
Problema con el testing

### DISCUSSION THREAD

**Discussion by Marco-Carballo-Francia** [atralice/Curso.Prep.Henry#44] [General]
¡Hola! Estoy haciendo la homework JS-I y el testing no me funciona de la manera adecuada. 
Después de instalar npm y poner el testing, esto es lo que me sale en la terminal:

No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In C:\Users\marco\OneDrive\Escritorio\Henry\Curso.Prep.Henry
  3 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 0 matches
  testPathIgnorePatterns: \\node_modules\\ - 3 matches
  testRegex:  - 0 matches
Pattern: JSI.test.js - 0 matches
npm ERR! Test failed.  See above for more details.

  Por lo que eh visto del video explicativo me debería de aparecer la cantidad de problemas y si están bien o mal pero en este caso no me aparece.

**Comment by rglvlncsr** [comment]
Marco, te puedes conectar por slack y  por ese medio me
gustaria colaborarle....lo primero es tener forkeado el repositorio...

**Comment by Marco-Carballo-Francia** [comment]
Como te encuentro en slack o por cual canal hablaremos? # 02_js1?
